spec.html

```diff
@@ -28366,7 +28367,7 @@ <h1>Static Semantics: ExportEntries</h1>
           1. Return a List whose sole element is _entry_.
         </emu-alg>
         <emu-note>
-          <p>*"\*default\*"* is used within this specification as a synthetic name for anonymous default export values.</p>
+          <p>*"\*default\*"* is used within this specification as a synthetic name for anonymous default export values. See <emu-xref href="#note-star-default-star">this note</emu-xref> for more details.</p>
```

Comment:
```suggestion
          <p>*"\*default\*"* is used within this specification as a synthetic name for anonymous default export values. See <emu-xref href="#note-star-default-star"></emu-xref> for more details.</p>
```

I don't like links that say things like "this" or "here".

Reply:
That note does not have a name or section number, so there's nothing which would be generated there. We need to put _some_ text there.

Reply:
Maybe `this note` → `a note on BoundNames`?

Reply:
I prefer the original text of "this note".